manifests/prometheus.yml

```diff
@@ -0,0 +1,547 @@
+---
+meta:
+  passthru: ((grab params.bosh_exodus_path || params.bosh || params.env))
+  default:
+    azs: [z1]
+  prometheus_port:   (( grab params.prometheus_port   || 8080 ))
+  grafana_port:      (( grab param.grafana_port       || 443 ))
+  alertmanager_port: (( grab params.alertmanager_port || 8082 ))
+
+exodus:
+  prometheus_url:   (( concat params.external_domain ":" meta.prometheus_port    ))
+  grafana_url:      (( concat params.external_domain ":" meta.grafana_port       ))
+  alertmanager_url: (( concat params.external_domain ":" meta.alertmanager_port  ))
+
+instance_groups:
+- name:      prometheus
+  instances: 1
+  azs:       (( grab params.availability_zones || meta.default.azs ))
+  stemcell:  default
+  persistent_disk_type: (( grab params.persistent_disk_type || "prometheus" ))
+  networks:   
+    - name: (( grab params.network || "default" ))
+      static_ips: (( grab params.static_ip ))
+  vm_type:   (( grab params.vm_type || "default" ))
+  jobs:
+  - name: alertmanager
```

Comment:
Put `release:` under `name:`, but before `properties:`.  In all cases, `properties:` should be the _last_ key because it is longwinded.